src/database/models/index.js

```diff
@@ -1,5 +1,5 @@
-/* eslint-disable global-require */
 /* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
```

Comment:
You are making so much noise on your PR by doing unnecessary changes. Better avoid having noisy PR.